### ISSUE TITLE
Consider a suitable presentation and management format for the EAS value lists

### ISSUE THREAD

**Issue by kerstarno** [SAA-SDT/EAS-Best-Practices#6]
## Creator of issue

1. Kerstin Arnold
2. EAD team lead, TS-EAS
3. @kerstarno
4. kerstin.arnold@archivesportaleurope.net

## The issue relates to

- [ ] EAC-CPF schema issue
- [ ] EAC-CPF Tag Library issue
- [ ] EAD schema issue
- [ ] EAD Tag Library issue
- [ ] Schema issue
- [ ] Tag Library issue
- [ ] Suggestions for all schemas
- [ ] Suggestions for all Tag Libraries
- [x] Other

## Wanted change/feature

* Text: This feature request follows https://github.com/SAA-SDT/eas-schemas/issues/1. Along with the decision to define the use of value lists via `@...Encoding` attributes within `<control>`, there was the decision to make sure the EAS value lists are defined somewhere outside of the schema, maybe also including some more descriptive information about what each value means and when it would be recommended to be used. There also should be a possibility for the community to suggest changes or additions, e.g. in the context of the annual minor revision cycles. While the conversation involved the option of exploring to use SKOS for this, it might also be worth considering whether this could become part of the Best Practices Guide.

**Comment by deletedname** [comment]
I agree the BPG is an appropriate place to provide a list of values for the relevant eas elements. Is there one I can put together to try some layout configurations?

**Comment by kerstarno** [comment]
If you wanted to, you could maybe use `@maintenanceEventType`. The current values are pretty straight-forward, I'd say, and we have a short description in the EAC-CPF 2.0 TL (https://eac.staatsbibliothek-berlin.de/schema/v2/eac.html#attr-maintenanceEventType) for each of them. 

Here are some initial thoughts re the aspects that we should/could include when presenting the EAS values:
- The value itself (obviously)
- A short description of the value
- A date for when the value and its description were created
- Prospectively the following aspects could be added:
  - A date for when the value and/or its description were updated
  - A short note about why the update was considered necessary
  - References to other vocabularies, if the values are either taken from another vocabulary or similar to those in another vocabulary

**Comment by rhyselia** [comment]
Also agree that the BPG is the most appropriate and useful. Thanks for testing out how this would look, Iris.

**Comment by deletedname** [comment]
After almost a year, I have an example of a value list on the BPG: [https://saa-sdt.github.io/EAS-Best-Practices/docs/value-lists](https://saa-sdt.github.io/EAS-Best-Practices/docs/value-lists). I've set up a couple more attributes in a separate branch called "additions-to-content": [audience](https://github.com/SAA-SDT/EAS-Best-Practices/blob/additions-to-content/docs/value-lists/audience.md) and [contactLineType](docs/value-lists/contactLineType.md). Please review and leave comments and suggestions :)

**Comment by kerstarno** [comment]
Hi Iris, 

thanks very much for this update. I've made some changes to [audience](https://github.com/SAA-SDT/EAS-Best-Practices/blob/additions-to-content/docs/value-lists/audience.md) and [contactLineType](https://github.com/SAA-SDT/EAS-Best-Practices/issues/docs/value-lists/contactLineType.md) with regard to the information about their expected use in EAD 4.0.

As for the example in the BPG:

- The text on the overview page needs amending, I think, to capture how EAD 4.0 is expected to use these value lists:
  - Value lists are provided as controlled terms defined ~within the~ by TS-EAS ~standards~. EAC-CPF 2.0 has these value lists defined in the schema; EAD 4.0 asks for an indication within `<control>` as to whether the EAS lists are used or whether the values are taken from another list. When elements or attributes have constrained data values, i.e. in EAC-CPF 2.0 or when EAD 4.0 is used with "EASList" as the encoding source, choose the appropriate term as defined in the EAS tag libraries. Some examples can be found below. 
- As for the example of `@addressLineType`, the information should be presented in the same way as the examples for `@audience` and `@contactLineType`, i.e. with a distinction between the uses in EAC-CPF 2.0 and EAD 4.0, I'd think.

**Comment by deletedname** [comment]
See Issue #16 -- value lists published to [https://saa-sdt.github.io/EAS-Best-Practices/docs/control/value-lists](https://saa-sdt.github.io/EAS-Best-Practices/docs/control/value-lists)
